docs/content/en/docs/reference/eksctl/anywhere_create_cluster.md

```diff
@@ -22,15 +22,15 @@ anywhere create cluster -f <cluster-config-file> [flags]
       --control-plane-wait-timeout string   Override the default control plane wait timeout (default "1h0m0s")
       --external-etcd-wait-timeout string   Override the default external etcd wait timeout (default "1h0m0s")
   -f, --filename string                     Filename that contains EKS-A cluster configuration
-      --force-cleanup                       Force deletion of previously created bootstrap cluster
   -z, --hardware-csv string                 Path to a CSV file containing hardware data.
   -h, --help                                help for cluster
       --install-packages string             Location of curated packages configuration files to install to the cluster
       --kubeconfig string                   Management cluster kubeconfig file
       --no-timeouts                         Disable timeout for all wait operations
-      --node-startup-timeout string         Override the default node startup timeout (default "10m0s")
+      --node-startup-timeout string         Override the default node startup timeout (Defaults to 20m for Tinkerbell clusters) (default "10m0s")
```

Comment:
This is because we didn't regenerate the docs after changing this flag
Same with the others that are not force-cleanup